catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -664,4 +729,22 @@ public void updateUserVariableReferencesInUserVariableBricks(List<UserVariable>
 			}
 		}
 	}
+
+	public void setConvertToSingleSprite(boolean convertToSingleSprite) {
```

Comment:
Ok, if you use it another way thats perfectly fine, I was just too lazy to look up if it is used another way ;)